src/Window.cpp

```diff
@@ -62,44 +58,53 @@ Window::Window(QWidget* parent)
   this->setAttribute(Qt::WA_NoSystemBackground, true);
   this->setAttribute(Qt::WA_TranslucentBackground, true);
 
+
+  // handle l_game_start_frame mouse tracking, because it sits at the top of Z-axis
   this->_masterFrameUI.l_game_start_frame->setMouseTracking(true);
   this->_masterFrameUI.l_game_start_frame->installEventFilter(this);
 
-  connect(this->game_start_movie, SIGNAL(frameChanged(int)), this, SLOT(handle_frame_changed(int)));
+  // to be able to stop the animation cleanly at frame 0
+  connect(this->_gameStartMovie, SIGNAL(frameChanged(int)), this, SLOT(handle_frame_changed(int)));
 }
 
-void Window::mousePressEvent(QMouseEvent *event)
+void Window::mousePressEvent(QMouseEvent* event)
 {
   _windowDragActive = true;
   _mouseEventPos = event->pos();
 }
 
-void Window::mouseReleaseEvent(QMouseEvent *event)
-{
-  _windowDragActive = false;
-}
+void Window::mouseReleaseEvent(QMouseEvent* event) { _windowDragActive = false; }
 
-void Window::mouseMoveEvent(QMouseEvent *event)
+void Window::mouseMoveEvent(QMouseEvent* event)
 {
   if (!_windowDragActive)
     return;
 
   move(event->globalPosition().toPoint() - _mouseEventPos);
 }
 
-bool Window::eventFilter(QObject *object, QEvent *event)
+bool Window::eventFilter(QObject* object, QEvent* event)
 {
   if (object == this->_masterFrameUI.l_game_start_frame && (event->type() == QEvent::MouseMove))
```

Comment:
So what does this whole thing do? Documentation.

Reply:
self documenting code